### ISSUE TITLE
devDependencies: uglifyify is missing

### ISSUE THREAD

**Issue by Arne-Pfeilsticker** [chiasm-project/chiasm-charts#1]
I have just tested the project and found out:
uglifyify is missing in the package.json.
"npm install http-server -g" needs to be installed.
Bay the way I want to say that I'm waiting eagerly for the layout for Angular Material. (To have the charts in different flexboxes.)

**Comment by curran** [comment]
Hi Arne,

Thanks for this note. I'm glad you have taken a look at this. I just added these as devDependencies.

I realized that it should be possible to use multiple instances of the existing Chiasm layout plugin (without modification) to inject visualizations into elements managed by CSS. Here's one example of this in action: [Bootstrap + Chiasm](http://bl.ocks.org/curran/bfc75d0fcc68b2d341f5). It's a bit verbose, but I think it pretty much does what you are looking for. Maybe we can use this as a starting point for a more clean solution, maybe an alternative to chiasm-layout like "chiasm-injector" that will inject multiple charts into multiple existing divs.
